src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -116,7 +116,7 @@
     // use by Google-written clients.
     func (c *{@view.clientTypeName}) SetGoogleClientInfo(name, version string) {
         goVersion := strings.Replace(runtime.Version(), " ", "_", -1)
-        v := fmt.Sprintf("%s/%s %s gax/%s go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
+        v := fmt.Sprintf("%s/%s %s gax/%s gl-go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
```

Comment:
I'll fix the `gax/{version}` duplication. But to answer the remaining question, under the new logic
- handwritten clients can just call `client.SetGoogleClientInfo("gccl", version)`
- the "constructor" of the client uses the same method to set `"gapic"/gapicVersion` tuple

I'll wait for the update on the internal doc (5pm Pacific) then send another revision